controllers/control/k8ssandratask_controller.go

```diff
@@ -200,6 +210,92 @@ func (r *K8ssandraTaskReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	}
 }
 
+func (r *K8ssandraTaskReconciler) executeRefreshTask(ctx context.Context, kTask *api.K8ssandraTask, kc *k8capi.K8ssandraCluster) (ctrl.Result, error) {
```

Comment:
[thought] It really bothers me to add an entirely new way to execute a task just for this.
- do we really need a task just to set an annotation on the cluster?
- is there any way we could handle DCs as part of the reconcile loop? That is, if we process a K8ssandraCluster with auto-update=once, then we propagate it to any DC with RequiresUpdate.